.github/workflows/build.yml

```diff
@@ -398,7 +398,7 @@ jobs:
     - name: "Collect Artifacts Metadata & Write Manifest"
       # Retrieve the metadata from the matrix job's outputs, merge them into a
       # single JSON document and then deploy to the server.
-      if: github.event_name == 'push' && env.SSH_PASSWORD != null
+      if: github.event_name == 'push'
```

Comment:
What is the point of building the manifest without deploying it? Just for testing right now?

Reply:
Doesn't hurt, that step takes far less than a second and it helps to catch errors and makes it easier to review the manifest output if someone makes a PR that changes the python script or the GItHub Actions workflow.

Reply:
We could even remove the `if:` condition for the whole "prepare for deployment" step, but I don't really care.

Reply:
I was wrong, it does hurt because it checks if the files in the manifest actually exist and fails otherwise. I'll remove the change.